<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmulatorSettingsWidget</class>
 <widget class="QWidget" name="EmulatorSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_6">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QCheckBox" name="fullscreen">
       <property name="text">
        <string>Boot apps in full screen</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="audio_backend"/>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Audio Backend</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="enable_ngs">
       <property name="text">
        <string>Enable NDS support</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QComboBox" name="log_level"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Log Level</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QCheckBox" name="archieve_log">
         <property name="text">
          <string>Archieve Log</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="discord_rte">
         <property name="text">
          <string>Discord Rich Presence</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QCheckBox" name="texture_cache">
         <property name="text">
          <string>Texture Cache</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="show_compile_shaders">
         <property name="text">
          <string>Show Compile Shaders</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QCheckBox" name="show_touchpad_cursor">
         <property name="text">
          <string>Show Touchpad Cursor</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="log_compatibility_warnings">
         <property name="text">
          <string>Log Compatibility Warnings</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Performance Overlay</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QCheckBox" name="performance_overlay">
          <property name="text">
           <string>Performance Overlay</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Emulated System Storage Folder</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="current_emu_folder">
            <property name="text">
             <string>Current emulator folder:</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="change_emulator_path">
              <property name="text">
               <string>Change Emulator Path</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="reset_emulator_path">
              <property name="text">
               <string>Reset Emulator Path</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>Custom Config Settings</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="clear_custom_config">
          <property name="text">
           <string>Clear Custom Config</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
